test/index.js

```diff
@@ -6,14 +6,38 @@ const resumer = require('resumer');
 const listen = require('test-listen');
 const micro = require('../lib/server');
 
-const { send, sendError, json } = micro;
+const {send, sendError, json, query} = micro
 
 const getUrl = fn => {
   const srv = micro(fn);
 
   return listen(srv);
 };
 
+test('support query string', async t => {
+  const qs = {test: 'test'}
+
+  const fn = async req => {
+    const q = query(req)
+    t.deepEqual(q, qs)
```

Comment:
With latest fix (7.2.2) – you need to return here something, otherwise request will hang.